public/index.html

```diff
@@ -1 +1,201 @@
-<meta http-equiv="refresh" content="0; url=/gdc/">
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
```

Comment:
I don't see any `no-js` or IE-specific CSS being used anywhere, but I suppose this is fine to keep